packages/paste-theme/src/utils/isDeprecatedTheme.ts

```diff
@@ -5,7 +5,7 @@ export const isDeprecatedTheme = (theme: DeprecatedThemeVariants): null => {
     console: 'It was removed in the 2022.02.04 release of Paste.',
   };
 
-  if (themeToDeprecationMessage[theme]) {
+  if (themeToDeprecationMessage[theme] && (process.env.NODE_ENV === 'development' || process.env.NODE_ENV === 'test')) {
```

Comment:
We write a test to prove the console error is printed so we have to check for test environments here for the console error to fire.